jdaviz/core/loaders/importers/image/image.py

```diff
@@ -71,6 +71,16 @@ def default_viewer_reference(self):
         # only used if `show_in_viewer=True` and no existing viewers can accept the data
         return 'imviz-image-viewer'
 
+    def _glue_data_wcs_to_fits(self, glue_data):
+        """
+        Re-set data.coords to a SIP approximation of the GWCS, if
+        gwcs_to_fits_sip is True. If this approximation is not possible, a
+        warning will be emitted and the original GWCS will be used.
+        """
+
+        if glue_data.coords is not None and self.gwcs_to_fits_sip:
```

Comment:
should we check to make sure the input is gwcs first (and probably only show the switch if so)?  Otherwise this raises an `AttributeError` that isn't handled in the existing try/except within `_try_gwcs_to_fits_sip`.